### ISSUE TITLE
Command test

### ISSUE THREAD

**Issue by whilefoo** [ubiquibot-whilefoo-testing/testing#7]
This issue is for testing the new command interface

**Comment by whilefoo** [comment]
@UbiquityOS assign me

**Comment by whilefoo** [comment]
@UbiquityOS i want to start this task

**Comment by whilefoo** [comment]
@UbiquityOS whats my wallet?

**Comment by whilefoo** [comment]
@UbiquityOS can you tell me @whilefoo2 wallet?

**Comment by whilefoo** [comment]
@UbiquityOS set my wallet to 0xF01704E94C61e18C47c969b56A4621AB08171E3A

**Comment by whilefoo** [comment]
@UbiquityOS allow @whilefoo2 to set priority labels

**Comment by whilefoo** [comment]
@UbiquityOS unassign me

**Comment by whilefoo** [comment]
@UbiquityOS what is the purpose of this repository?

**Comment by gentlementlegen** [comment]
@UbiquityOS can you help me?

**Comment by gentlementlegen** [comment]
@UbiquityOS assign @whilefoo

**Comment by gentlementlegen** [comment]
@UbiquityOS run the /start command for user @whilefoo

**Comment by gentlementlegen** [comment]
@UbiquityOS I want to start this task

**Comment by whilefoo** [comment]
@UbiquityOS run the /start command for user @gentlementlegen

**Comment by whilefoo** [comment]
> @UbiquityOS run the /start command for user @whilefoo

I don't think this would work correctly because you can only run /start on yourself and additional teammates otherwise anybody could assign someone since we don't have access control for this

**Comment by gentlementlegen** [comment]
@whilefoo of course, I was just testing if you could run unintended commands like un-assigning other people. But the `/start` for myself should have run isn't it?

**Comment by whilefoo** [comment]
test

**Comment by whilefoo** [comment]
test

**Comment by whilefoo** [comment]
/@whilefoo test

**Comment by whilefoo** [comment]
/@whilefoo test

**Comment by whilefoo** [comment]
/@whilefoo test

**Comment by whilefoo** [comment]
/@EresDevOrg test

**Comment by whilefoo** [comment]
/@whilefoo test
